code/defines/procs/admin.dm

```diff
@@ -68,7 +68,10 @@
 	return .
 
 /proc/key_name_admin(whom)
-	var/message = "[key_name(whom, 1)](<A HREF='?_src_=holder;adminmoreinfo=\ref[whom]'>?</A>)[isAntag(whom) ? "<font color='red'>(A)</font>" : ""][isLivingSSD(whom) ? "<span class='danger'>(SSD!)</span>" : ""] ([admin_jump_link(whom)])"
+	var/mob/whom_mob = whom //Should be always be a mob.
+	if(!istype(whom_mob)) //Best to check anyway
+		return  "key_name_admin did get a non-mob as argument. Argument is [whom]. Report it to a coder."
```

Comment:
Can you change the phrasing to "Make an issue report or report it to a coder" - I am sick of people constantly pinging me and coders for this.